yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/redundant-indexes.psql

```diff
@@ -0,0 +1,293 @@
+
+CREATE TEMP TABLE ind_att AS 
```

Comment:
can temp tables have same name as one of the existing table?

If not you might want to handle the name for tables created here.

Reply:
It is allowed to create temp tables with the same name as an existing table.
```
postgres=# CREATE TABLE temp_table(id int);
CREATE TABLE
postgres=# CREATE TEMP TABLE temp_table(id int);
CREATE TABLE
postgres=# DROP TABLE temp_table;
DROP TABLE
postgres=# \d
postgres=# \dt temp_table 
           List of relations
 Schema |    Name    | Type  |  Owner   
--------+------------+-------+----------
 public | temp_table | table | postgres
(1 row)

postgres=# \d temp_table 
             Table "public.temp_table"
 Column |  Type   | Collation | Nullable | Default 
--------+---------+-----------+----------+---------
 id     | integer |           |          | 

postgres=# 
```